packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -456,11 +455,6 @@ describe("AuthCodeRedirectHandler", () => {
       await authCodeRedirectHandler.handle(
         "https://coolsite.com/redirect?code=someCode&state=oauth2StateValue"
       );
-      // Check that the current session is stored correctly __specifically__ in
-      // 'localStorage'.
-      expect(window.localStorage.getItem(KEY_CURRENT_SESSION)).toBe(
```

Comment:
Why isn't this check needed? Should the assertion be covered in a different unit test?

Reply:
That's right, it has been moved from `AuthCodeRedirectHandler` to `Session`. I'll cover this in an appropriate unit test.

Reply:
Fixed in 124c004311e95d3de470bc408732cfc469cb1d80